lib/msf/core/optional_session/ldap.rb

```diff
@@ -5,8 +5,8 @@ module OptionalSession
     module LDAP
       include Msf::OptionalSession
 
-      RHOST_GROUP_OPTIONS = %w[RHOSTS RPORT DOMAIN USERNAME PASSWORD THREADS]
-      REQUIRED_OPTIONS = %w[RHOSTS RPORT THREADS]
+      RHOST_GROUP_OPTIONS = %w[RHOSTS RPORT LDAPDomain LDAPUsername LDAPPassword THREADS]
+      REQUIRED_OPTIONS = %w[RHOSTS RPORT LDAPUsername LDAPPassword THREADS]
```

Comment:
Following the convention from PR: https://github.com/rapid7/metasploit-framework/pull/19551
```suggestion
      REQUIRED_OPTIONS = %w[RHOSTS RPORT THREADS]
```